sources/packages/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -104,12 +107,10 @@ export class RestrictionController extends BaseController {
    * @param restrictionCategory Selected Category from category list.
    * @returns Reasons option list.
    */
-  @Groups(UserGroups.AESTUser)
-  @AllowAuthorizedParty(AuthorizedParties.aest)
   @Get("/reasons/options-list/category/:restrictionCategory")
   async getReasonsOptionsList(
     @Param("restrictionCategory") restrictionCategory: string,
```

Comment:
`restrictionCategory` must be validated. It is a list of known values, right?

Reply:
It is a String of 50 maxlength. We are not validating in general.